src/jvm/main/org/jetbrains/kotlinx/lincheck/Reporter.kt

```diff
@@ -199,22 +210,55 @@ internal class TableLayout(
 internal fun ExecutionLayout(
     initPart: List<String>,
     parallelPart: List<List<String>>,
-    postPart: List<String>
+    postPart: List<String>,
+    validation: List<String>?
```

Comment:
Please make the absence of the validation consistent with other execution parts: use emptyList instead of null